pkg/querier/querier.go

```diff
@@ -49,6 +49,9 @@ type Config struct {
 	QueryIngestersWithin      time.Duration `yaml:"query_ingesters_within"`
 	EnablePerStepStats        bool          `yaml:"per_step_stats_enabled"`
 
+	// Use compression when returning promql response. Supported values are 'gzip' or '' (disable compression)
```

Comment:
We also support `snappy` as mentioned in the flag below. I suggest removing the comment here so we only maintain the help text in flag